hack/generate-manifest.sh

```diff
@@ -201,8 +201,9 @@ if [ "$TUN_TYPE" != "geneve" ] && [ "$TUN_TYPE" != "vxlan" ] && [ "$TUN_TYPE" !=
     exit 1
 fi
 
-if [ "$MODE" == "release" ] && [ -z "$IMG_NAME" ]; then
-    echoerr "In 'release' mode, environment variable IMG_NAME must be set"
+if [ "$MODE" == "release" ] && [ -z "$AGENT_IMG_NAME" ] && [ -z "$CONTROLLER_IMG_NAME" ]; then
```

Comment:
What will happen if `AGENT_IMG_NAME` is set and `CONTROLLER_IMG_NAME` is unset? I suppose you want both of them be set, right?

Reply:
yes that's why i have used && condition.

Reply:
@jainpulkit22 No, if you use && condition, it will only exit when both values are unset.

Reply:
yes got this have updated the code, am just pushing the changes.